README.md

```diff
@@ -1 +1,75 @@
-# Stock-Market
\ No newline at end of file
+# Stock-Market
+
+## Deployment
+
+(Link will be provided after finishing the project)
+
+## Major Languages
+
+- HTML
+- CSS
+- JavaScript
+- React JS
+- Redux
+- API
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+To get a local copy up and running follow these simple steps.
+
+To setup Stock-market project, in the repo page:
+click on code (dropdown list) > Download as ZIP;
+or open terminal of path you want to install project and run this command <br>
+
+```sh
+  git clone git@github.com:MohammedElseddik/Stock-Market.git
+```
+
+## 🛠 Installation & Set Up to run the project locally
+
+1. Install Node Modules
+
+   ```sh
+   npm install
+   ```
+
+2. Start the development server
+
+   ```sh
+   npm start
+   ```
+
+the project will be opened in [http://localhost:3000] on the browser.
+
+## Authors
+
+👤 **Mohammed Elseddik Rebbouh**
+
+- GitHub: [@githubhandle](https://github.com/MohammedElseddik)
+- Twitter: [@twitterhandle](https://twitter.com/RebbouhElseddik)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammed-elseddik-rebbouh-676500192)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/MohammedElseddik/bookstore/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- My coding partners
+- Hat tip to anyone whose code was used.
+- Microverse team.
+- My standup team.
+
```

Comment:
- [ ] Kindly make sure you add you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the **Acknowledgments** Section

Reply:
This issue is well resolved! ✔️